### ISSUE TITLE
precedence with <meta name="color-scheme" ...>

### ISSUE THREAD

**Issue by jasonwilliams** [WICG/web-preferences-api#34]
Should this specification discuss the precedence with the color-scheme [meta tag](https://html.spec.whatwg.org/multipage/semantics.html#meta-color-scheme)?

I assume the preferences API will override the color scheme tag (as its user driven) but it isn't mentioned in the docs/spec. I don't know if this would require an update to the color-scheme meta tag section?

My assumptions for now is:
Web Prefs Override -> color-scheme meta tag -> OS default

**Comment by lukewarlow** [comment]
Off the top of my head I think it works with the meta tag rather than overriding. So you'd say you support both values and then the API can switch which value is chosen. Just like you today say you support both values and the browser picks the correct value based on the OS.

**Comment by jasonwilliams** [comment]
Sure, that makes sense when you pass both `dark light` into the meta tag, but what happens when you only pass one?

```html
<meta name="color-scheme" content="light only">
```

```js
await navigator.preferences.colorScheme.requestOverride("dark")
```

**Comment by lukewarlow** [comment]
Yeah the case with a single value is interesting, and the case with a single value and only is potentially even more interesting. Needs thinking about. I think in all cases the media query would update at least. But potentially the used colour scheme wouldn't?

Similar to https://github.com/WICG/web-preferences-api/issues/14

**Comment by lukewarlow** [comment]
Moved to https://github.com/w3c/csswg-drafts/issues/11576
